### PR TITLE
Fixed typo in kernel package name.

### PR DESCRIPTION
**Pull request by Dmitry-Eremeev**
Tested package in CentOS 8

### CODE REVIEW

**Comment by elguero** [comment]
@Dmitry-Eremeev Thanks for the contribution.

Looking through past issues, #61, the typo fix should be fine.

Adding CentOS 8 support doesn't appear to be as simple as just adding it to Travis to create an RPM.  Please look at #113, #122 and DisplayLink/evdi#193.

If you want to update this to only change the kernel package name, that should be fine.

**Comment by Dmitry-Eremeev** [comment]
> @Dmitry-Eremeev Thanks for the contribution.
> 
> Looking through past issues, #61, the typo fix should be fine.
> 
> Adding CentOS 8 support doesn't appear to be as simple as just adding it to Travis to create an RPM. Please look at #113, #122 and [DisplayLink/evdi#193](https://github.com/DisplayLink/evdi/issues/193).
> 
> If you want to update this to only change the kernel package name, that should be fine.

Hi,

I removed CentOS 8 from CI.
JIC it works in my CentOS desktop.
For those who's struggling with CentOS 8:
    
    rpm --import https://www.elrepo.org/RPM-GPG-KEY-elrepo.org
    
    dnf install https://www.elrepo.org/elrepo-release-8.el8.elrepo.noarch.rpm
    
    dnf --enablerepo=elrepo-kernel install kernel-ml kernel-ml-devel
    
    dnf install rpm-build gcc-c++ libdrm-devel epel-release dkms kernel-headers kernel-devel make redhat-lsb-core
    
    dkms install evdi/1.7.0
    
    cd displaylink-rpm/
    
    make
    
    dnf install x86_64/displaylink-1.7.0-1.x86_64.rpm

File: displaylink.spec [hunk 1]
```diff
@@ -167,6 +167,9 @@ chmod +x %{buildroot}%{_prefix}/lib/systemd/system-sleep/displaylink.sh
 %systemd_postun_with_restart displaylink.service
 
 %changelog
+* Sat May 30 2020 Mitya Eremeev <mitossvyaz@mail.ru>
+- fix typo in kernel package name
+- tested package in CentOS 8
```
Comment (elguero) [inline displaylink.spec:172]: You should add `1.7.0-2` to the end of the line with the date and author information.

Comment (elguero) [inline displaylink.spec:172]: Add a space between this changelog entry and the one below it.

**Pull request merged**